hazelcast-hibernate53/src/test/java/com/hazelcast/hibernate/local/TestLocalCacheRegionFactory.java

```diff
@@ -0,0 +1,35 @@
+package com.hazelcast.hibernate.local;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.hibernate.HazelcastLocalCacheRegionFactory;
+import org.hibernate.boot.spi.SessionFactoryOptions;
+import org.hibernate.cache.spi.CacheKeysFactory;
+
+import java.util.Map;
+
+/**
+ * This class just extends HazelcastLocalCacheRegionFactory to allow using a CleanupService with a shorter delay.
+ */
+public class TestLocalCacheRegionFactory extends HazelcastLocalCacheRegionFactory {
+
+    // Visible for testing
+    public final static long CLEANUP_PERIOD = 12L;
```

Comment:
Could it even be shorter than 12 seconds?

Reply:
I've reduced it to 6 seconds.